src/delete.ts

```diff
@@ -7,8 +7,8 @@ export const del = async (
 	prefix: string,
 	suffix: string,
 	version: string
-): Promise<string> => {
-	const config = await startup();
+): Promise<void> => {
```

Comment:
```suggestion
): Promise<string> => {
```

is it required to keep it `void`?